src/graphql/resolvers/businessInfoResolver.ts

```diff
@@ -6,6 +6,52 @@ import {
 import { getInterpretationProvider, InterpretationConfig } from '../../providers/interpretation_providers';
 import { BusinessInferences, QueryGetBusinessInfoArgs } from '../generatedTypes';
 import { defaultConfig, DataSourceConfiguration } from '../../models/data_models/dataSourceConfiguration';
+import { BusinessInfoInput } from '../../models/data_models/types';
+import { GraphQLClient, gql } from 'graphql-request';
+
+async function fetchFromDB(
+	endpoint: string,
+	businessInfo: BusinessInfoInput
+): Promise<BusinessInferences | undefined> {
```

Comment:
You are mixing two different types of types here. `BusinessInferences` refers to the type generated from the GraphQL schema, while `BusinessInfoInput` refers to the internal type that isn't associated with the schema. You likely mean to use the `BusinessInferences` type defined in `../../models/insight.ts`. This issue isn't apparent because these two `BusinessInferences` types look extremely similar.

To prevent the confusion from happening in the future, the BusinessInferences type generated from the schema should be imported as another name (such as `QueryBusinessInferences`), and `BusinessInferences` can be imported from `insight.ts`.